config/core/resources/eventtype.yaml

```diff
@@ -20,6 +20,7 @@ metadata:
     knative.dev/crd-install: "true"
 spec:
   group: eventing.knative.dev
+  preserveUnknownFields: false
```

Comment:
The addition of this with the lack of defining a status object in the schema means status updates are dropped.